packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -93,9 +93,11 @@ export default class FreeDrawLayer extends PureComponent {
   }
 
   endDraw() {
-    Object.keys(this.events).forEach(name => {
-      removeEvent(this.events[name])
-    })
+    if (this.events) {
```

Comment:
would it be better to check for keys here instead?

Reply:
What do you mean? The only reason I changed this was because jest / babel complained about this line when this.events in undefined.

Reply:
I just mean check if `this.events` actually contains anything, but ok